dev-python/elasticsearch-curator/elasticsearch-curator-5.8.1-r3.ebuild

```diff
@@ -0,0 +1,165 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_7 )
+
+MY_PN="curator"
+ES_VERSION="7.3.2"
+
+inherit distutils-r1
+
+DESCRIPTION="Tending time-series indices in Elasticsearch"
+HOMEPAGE="https://github.com/elastic/curator"
+SRC_URI="https://github.com/elastic/${MY_PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz
+	test? ( https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-${ES_VERSION}-linux-x86_64.tar.gz )"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+# tests fail in chroot
+# https://github.com/elastic/elasticsearch/issues/12018
+RESTRICT="test"
+IUSE="doc test"
+
+# vulnerable pyyaml
+# https://github.com/elastic/curator/issues/1415
+RDEPEND="
+	>=dev-python/elasticsearch-py-7.0.4[${PYTHON_USEDEP}]
+	<dev-python/elasticsearch-py-8.0.0[${PYTHON_USEDEP}]
+	>=dev-python/click-6.7[${PYTHON_USEDEP}]
+	<dev-python/click-7.0[${PYTHON_USEDEP}]
+	>=dev-python/certifi-2019.9.11[${PYTHON_USEDEP}]
+	>=dev-python/requests-2.20.0[${PYTHON_USEDEP}]
+	>=dev-python/urllib3-1.24.2[${PYTHON_USEDEP}]
+	<dev-python/urllib3-1.26[${PYTHON_USEDEP}]
```

Comment:
Try without downgrading urllib3

Reply:
Would like to use a current urllib3 version. But this is a requirement of elasticsearch curator. Please see this commit: https://github.com/elastic/curator/commit/99bb6d46823ac33fed399f5b0786e33dee2e9058

Reply:
As compilation with dev-python/urllib3-1.26.2-r3 works here, can you find out what's upsteam's intention with pinning that dependency?

Reply:
https://github.com/elastic/curator/issues/1589

Reply:
Could you also comment this on https://bugs.gentoo.org/714860 please?